frontend/src/components/object/ObjectProperties.vue

```diff
@@ -44,21 +42,10 @@ async function load() {
   bucket.value = bucketStore.findBucketById(object.value?.bucketId as string);
 
   if( props.fullView ) {
-    if( props.versionId ) {
-      version.value = versionStore.findVersionById(props.versionId);
-      objectMetadata.value = versionStore.findMetadataByVersionId(props.versionId);
-
-      await userStore.fetchUsers({userId:[version.value?.createdBy, version.value?.updatedBy]});
-      createdBy.value = getUserSearch.value.find( x => x.userId === version.value?.createdBy )?.fullName;
-      updatedBy.value = getUserSearch.value.find( x => x.userId === version.value?.updatedBy )?.fullName;
-    }
-    else {
-      objectMetadata.value = metadataStore.findMetadataByObjectId(object.value?.id as string);
-
-      await userStore.fetchUsers({userId:[object.value?.createdBy, object.value?.updatedBy]});
-      createdBy.value = getUserSearch.value.find( x => x.userId === object.value?.createdBy )?.fullName;
-      updatedBy.value = getUserSearch.value.find( x => x.userId === object.value?.updatedBy )?.fullName;
-    }
+    objectMetadata.value = metadataStore.findMetadataByObjectId(object.value?.id as string);
+    await userStore.fetchUsers({userId:[object.value?.createdBy, object.value?.updatedBy].filter((userId) => userId != SYSTEM_USER)});
```

Comment:
Lining - this line must be broken down:
```ts
await userStore.fetchUsers({
  userId: [object.value?.createdBy, object.value?.updatedBy]
    .filter((userId) => userId !== SYSTEM_USER)
});
```
Looking further at this, it might not be a bad idea to consider folding in that `SYSTEM_USER` filter into the fetchUsers store method instead if appropriate?

Reply:
filtering out SYSTEM_USER is now working in the service layer, so i've removed that filter() here